app/core/storage.js

```diff
@@ -1,9 +1,20 @@
+import { ipcRenderer } from 'electron'
```

Comment:
The storage module needed to be refactored for some reason the default path it was using changed during the electron update. Its working exactly as expected with these changes